app/src/sharedTest/java/org/oppia/android/app/player/audio/AudioFragmentTest.kt

```diff
@@ -190,91 +203,140 @@ class AudioFragmentTest {
         .check(matches(withContentDescription(context.getString(R.string.audio_play_description))))
     }
   }
 
   @Test
-  // TODO(#973): Fix AudioFragmentTest
-  @Ignore
   fun testAudioFragment_invokePrepared_clickPlay_touchSeekBar_checkStillPlaying() {
+    testMediaPlayer.addMediaInfo(
+      context = context,
+      explorationId1 = RATIOS_EXPLORATION_ID_0,
+      explorationId2 = RATIOS_EXPLORATION_ID_0,
+      audioFileName1 = audioFileName1,
+      audioFileName2 = audioFileName2,
+      mediaPlayer = audioPlayerController.getTestMediaPlayer()
+    )
     launch<AudioFragmentTestActivity>(
-      createHomeActivityIntent(
+      createAudioFragmentTestIntent(
         PROFILE_ID_DEFAULT_AUDIO_LANGUAGE_ENGLISH
       )
     ).use {
-      invokePreparedListener(shadowMediaPlayer)
+      testCoroutineDispatchers.runCurrent()
 
       onView(withId(R.id.ivPlayPauseAudio)).perform(click())
+      testCoroutineDispatchers.runCurrent()
       onView(withId(R.id.sbAudioProgress)).perform(clickSeekBar(100))
 
+      testCoroutineDispatchers.runCurrent()
       onView(withId(R.id.ivPlayPauseAudio))
         .check(matches(withContentDescription(context.getString(R.string.audio_pause_description))))
     }
   }
 
-  @Test
-  @Ignore("Landscape not properly supported") // TODO(#56): Reenable once landscape is supported.
-  fun testAudioFragment_invokePrepared_playAudio_configurationChange_checkStillPlaying() {
-    launch<AudioFragmentTestActivity>(
-      createHomeActivityIntent(
-        PROFILE_ID_DEFAULT_AUDIO_LANGUAGE_ENGLISH
-      )
-    ).use {
-      invokePreparedListener(shadowMediaPlayer)
-      onView(withId(R.id.ivPlayPauseAudio)).perform(click())
-      onView(withId(R.id.sbAudioProgress)).perform(clickSeekBar(100))
-      onView(isRoot()).perform(orientationLandscape())
-      onView(withId(R.id.ivPlayPauseAudio))
-        .check(matches(withContentDescription(context.getString(R.string.audio_pause_description))))
-    }
-  }
-
+  /**
+   * TODO: Remove the comment when fixing this test case
+   * Commenting this test case as we are no loonger using below method:
+   * invokePreparedListener(shadowMediaPlayer)
+   *
+   * Try to use TestMediaPlayer
+   */
+  /* @Test
+   @Ignore("Landscape not properly supported") // TODO(#56): Reenable once landscape is supported.
+   fun testAudioFragment_invokePrepared_playAudio_configurationChange_checkStillPlaying() {
+     launch<AudioFragmentTestActivity>(
+       createAudioFragmentTestIntent(
+         PROFILE_ID_DEFAULT_AUDIO_LANGUAGE_ENGLISH
+       )
+     ).use {
+       invokePreparedListener(shadowMediaPlayer)
+       onView(withId(R.id.ivPlayPauseAudio)).perform(click())
+       onView(withId(R.id.sbAudioProgress)).perform(clickSeekBar(100))
+       onView(isRoot()).perform(orientationLandscape())
+       onView(withId(R.id.ivPlayPauseAudio))
+         .check(matches(withContentDescription(context.getString(R.string.audio_pause_description))))
+     }
+   }*/
```

Comment:
Why exactly does this test need to be commented out? We shouldn't check in commented out code since it can easily get out of date.

Reply:
Actually, on line number 297, we have `invokePreparedListener(shadowMediaPlayer)` which is no longer a part of this particular file and need update but as this test is ignored, I can't update this test. 

Should I update this test?